### DISCUSSION TITLE
Status of socket.io-p2p

### DISCUSSION THREAD

**Discussion by Semro** [socketio/socket.io#3905] [General] [answered]
Hello. I would like to use [socket.io-p2p](https://github.com/socketio/socket.io-p2p) in my project but the repository was abandoned a long time ago. 
Will this ever be maintained in the future? Or at least to have some roadmap to make it compatible with the latest socket.io and meet modern standards.
It looks like a simple approach to make a p2p apps with socket.io and would be very useful.

**Comment by darrachequesne** [comment, marked as answer]
Hi, thanks for the heads-up.

The project is indeed currently not maintained, I've opened an issue about this here: https://github.com/socketio/socket.io-p2p/issues/72
